Library/Formula/knot.rb

```diff
@@ -65,11 +66,11 @@ def knot_conf; <<-EOS.undent
     log {
       syslog {
         any error;
-        zone warning, notice;
+        zone notice;
         server info;
       }
       stderr {
-        any error, warning;
+        any warning;
       }
     }
     EOS
```

Comment:
The regular "make install" actually installs a knot.sample.conf, so perhaps I could remove the "inreplace" call that disables this?